src/pages/MainPage.jsx

```diff
@@ -0,0 +1,105 @@
+import { Link } from "react-router-dom";
+import MainSection from "../components/MainSection";
+import logo from "../assets/images/logo.svg";
+import logoText from "../assets/images/logo_text.svg";
+import topImg from "../assets/images/home_top.svg";
+import bottomImg from "../assets/images/home_bottom.svg";
+import img1 from "../assets/images/home_01.svg";
+import img2 from "../assets/images/home_02.svg";
+import img3 from "../assets/images/home_03.svg";
+
+function MainPage() {
+  return (
+    <div>
+      <div className="w-full bg-white sticky top-0">
+        <div className="max-w-1120 py-10 px-16 flex items-center justify-between tablet:px-24 m-auto">
+          <div className="py-8 flex-1 tablet:py-0">
+            <img className="w-153 hidden tablet:block" src={logo} />
+            <img className="w-103 block tablet:hidden" src={logoText} />
+          </div>
+          <button>로그인</button>
+        </div>
+      </div>
+      <div className="bg-[#cfe5ff] flex items-end">
+        <div className="mx-auto flex flex-col items-center gap-132 tablet:gap-212 pc:gap-8 pc:flex-row pc:pt-200">
+          <div className="flex flex-col gap-18 tablet:gap-24 pc:gap-32 pt-48 tablet:pt-84 pc:pt-0">
+            <h2 className="text-center text-3xl font-bold text-gray700 pc:text-left">
+              일상의 모든 물건을 <br className="block tablet:hidden pc:block" />
+              거래해보세요
+            </h2>
+            <Link to="/items">
+              <button>구경하러 가기</button>
+            </Link>
+          </div>
+          <div>
+            <img className="h-172 tablet:h-340" src={topImg} />
+          </div>
+        </div>
+      </div>
+      <div className="flex flex-col gap-40 py-52 tablet:gap-52 pc:gap-138 pc:py-138">
+        <MainSection
+          img={img1}
+          badge="Hot Item"
+          title={
+            <>
+              인기 상품을 <br className="hidden pc:block" /> 확인해 보세요
+            </>
+          }
+          description={
+            <>
+              가장 HOT한 중고거래 물품을
+              <br />
+              판다 마켓에서 확인해 보세요
+            </>
+          }
+        />
```

Comment:
💊 제안
반복되는 형태의 UI를 공통으로 사용하기로 하셨다면 아래처럼 하시면 더 의도가 명확하고 좋을 것 같습니다.

```suggestion
        {contents.map(({ img, badge, title, content} ) => (
          <MainSection
            key={category}
            img={img}
            badge={badge}
            title={title}
            description={description}
          />
        ))}
```